modules/n1ql/pages/n1ql-language-reference/condfunnum.adoc

```diff
@@ -4,32 +4,38 @@
 
 {description}
 
-IFINF(expression1, expression2, \...)
+== IFINF(expression1, expression2, \...)
 
+=== Description
```

Comment:
Even though it is a well-known function, it would be great if we could add usage example per function